jwst/tweakreg/tweakreg_step.py

```diff
@@ -204,9 +205,7 @@ def process(self, input):
                 self.log.warning("At least two exposures are required for image "
                                  "alignment.")
                 self.log.warning("Nothing to do. Skipping 'TweakRegStep'...")
-                self.skip = True
-                for model in images:
-                    model.meta.cal_step.tweakreg = "SKIPPED"
+                record_step_status(input, "tweakreg", success=False)
```

Comment:
```suggestion
                record_step_status(images, "tweakreg", success=False)
                return images
```
I think the `return input` (which I can't seem to get to be part of the suggestion) is a pre-existing bug in this step but could be fixed as part of this PR.